public/intro/2_Lecture_2.md

```diff
@@ -82,7 +82,7 @@ Debugging is an important part of programming, this section will introduce you t
 
 ### Asserts
 
-We can use assert statements to check our assumptions in various places. When an assert statement fails, we get a precise location information. Asserts are generally only use during the development phase, and below you find an example on how to desactivate them with a macro for your production code. To use assert statements, you have to include the _assert.h_ library.
+We can use assert statements to check our assumptions in various places. When an assert statement fails during runtime, the program is stopped we get and error message with precise location information. Asserts are generally only used during the development phase, and below you find an example on how to deactivate them with a macro for your production code. To use assert statements, you have to include the _assert.h_ library.
```

Comment:
Typo on the "we get and error message"

Reply:
Done

Reply:
Also, you could really have done that by yourself bro. You’re really becoming a senior developer here